diagnostic_rt_qpcr/operation_types/aliquot_positive_template/protocol.rb

```diff
@@ -1,170 +1,162 @@
-# Aliquot Positive Template Protocol
+# Aliquot Positive Template
 # Written By Rita Chen 2020-05-04
+# Updated by Dany Fu 2020-06-01
+
+needs 'Standard Libs/LabwareNames'
 
 # 2019-nCoV Positive Control (nCoVPC) Preparation:
 # 1) Precautions: This reagent should be handled with caution in a dedicated
 # nucleic acid handling area to prevent possible contamination. Freeze-thaw
 # cycles should be avoided. Maintain on ice when thawed.
 # 2) Resuspend dried reagent in each tube in 1 mL of nuclease-free water to
 # achieve the proper concentration. Make single use aliquots (approximately 30
-# L) and store at less than and equal to -70C.
+# µL) and store at less than and equal to -70C.
 # 3) Thaw a single aliquot of diluted positive control for each experiment and
 # hold on ice until adding to plate. Discard any unused portion of the aliquot.
-
 class Protocol
+  include Units
+  include LabwareNames
+
+  VOL_WATER = { qty: 1, units: MILLILITERS }.freeze
+  VOL_SUSPENSION = { qty: 30, units: MICROLITERS }.freeze
+  OUTPUT_ITEMS_NUM = { qty: 33, units: TUBE_MICROFUGE }.freeze
 
   def main
     # 1. Get 33 1.5 mL tube for each operation
-    get_tubes(operations.length)
+    get_tubes(count: operations.length)
     operations.retrieve
 
     # 2. For each Lyophilized Postive Control, resuspend in 1 mL of
     # nuclease-free waterand, aliquot 33 use aliquots (approximately
-    # 30 L) and store at less than and equal to -70C.
+    # 30 µL) and store at less than and equal to -70C.
+    save_output(ops: operations)
+
+    suspend_lyophilized_RNA
+    keep_tubes = [] # Empty array for storing single use aliquots
 
     # Group the operations by the input reagent
-    ops_by_input = operations.group_by {|op| op.input("Template").item}
-    ops_by_input.each do |lyophilized_RNA, ops|
-      label_and_locate_output(ops)
+    ops_by_input = operations.group_by { |op| op.input('Template').item }
+    ops_by_input.each do |lyophilized_rna, ops|
+      keep_tubes.push(make_aliquots(ops: ops, lyophilized_rna: lyophilized_rna))
     end
 
-    suspend_lyophilized_RNA(operations)
-    keep_tubes = [] #Empty array for storing single use aliquots
-    
-    ops_by_input.each do |lyophilized_RNA, ops|
-      keep_tubes.push(make_aliquots(ops))
-    end
-    
-    prepare_plating(keep_tubes)
-    
+    prepare_plating(keep_tubes: keep_tubes)
+
     # 3. Thaw a single aliquot of diluted positive control for each
     # experiment and hold on ice until adding to plate.
     # Discard any unused portion of the aliquot.
-    operations.store(interactive: true, io: "output", method: "boxes")
+    operations.store(interactive: true, io: 'output', method: 'boxes')
   end
 
   # Get 33 1.5 mL tubes per dried positive control
   #
   # @param count [Integer] the number of operations
-  def get_tubes(count)
+  def get_tubes(count:)
     show do
-      title "Get new 1.5 mL tubes"
-      check "Please get #{count*33} 1.5 mL tubes"
+      title "Get new #{TUBE_MICROFUGE}"
```

Comment:
`TUBE_MICROFUGE.pluralize(count * OUTPUT_ITEMS_NUM[:qty])` will add an s if there is more than 1